src/app/clusters/ota-requestor/ota-downloader.h

```diff
@@ -0,0 +1,50 @@
+/*
+ *
+ *    Copyright (c) 2021 Project CHIP Authors
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+/* This file contains the declarations for the OTADownloader class which
+ * abstracts the image download functionality from the particular protocol
+ * used for it.
+ * Applications and platforms implementing the OTA Requestor functionality
+ * must include this file
+ */
+
+#include <ota-image-processor.h>
```

Comment:
I'm not sure if there are specific naming convention documented somewhere but I believe most file names are PascalCase or at the very least, something similar to BDXDownloader.h.

Reply:
Out of 853 header files under ./src at least 105 are in lower case:
>fd *.h | wc -l 
853
>find . -name *.h | grep -v [A-Z] | wc -l
105